index.html

```diff
@@ -28,9 +28,16 @@
     </main>
 
     <footer>
-      <div class="logo">Logo</div>
-      <div class="links">Links</div>
-      <div class="copyright">Copyright</div>
+      <div class="wrapper">
+        <div class="logo">Chris</div>
+        <div class="links">
```

Comment:
Is this your nav? If so I would suggest the nav tag

Reply:
> Is this your nav? If so I would suggest the nav tag

No. It is a simple footer :)